platform/src/pages/index.js

```diff
@@ -1,10 +1,17 @@
+import { NEXT_PLATFORM_ENV } from 'lib/envConstants';
+
 function HomePage() {
   return (
     <>
       <h1>Airqo platform (beta)</h1>
       <div>
         Find the original airqo platform
-        <a href={process.env.NEXT_PUBLIC_ORIGINAL_PLATFORM}>here</a>
+        <a
+          className='text-3xl p-2 underline font-bold'
+          href={NEXT_PLATFORM_ENV}
```

Comment:
Instead of calling process.env in the frontend, I fetch the env variable via a constant variable created in the envConstants file.